packages/client/wallets/aa/src/utils/blockchain.ts

```diff
@@ -1,27 +1,5 @@
-import { getBundlerRPC } from "@/blockchain";
-import { createPublicClient, http } from "viem";
-
 import { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
 
-export type VerifyMessageParams = {
-    address: `0x${string}`;
-    message: string;
-    signature: `0x${string}` | Uint8Array;
-    chain: EVMBlockchainIncludingTestnet;
-};
-
-export async function verifyMessage({ address, message, signature, chain }: VerifyMessageParams) {
```

Comment:
I think we want this, or at least ethers verify message didn't work with AAs, so we added this helper

Reply:
We expose a `public client` from `EVMSmartWallet` now, so `wallet.public.client.verifyMessage` would be equivalent.

Reply:
We gotta add a doc example though, I think that's super obvious

Reply:
going to merge this, can circle back before the release if you'd like